src/init.cpp

```diff
@@ -329,6 +329,8 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-alerts", strprintf(_("Receive and display P2P network alerts (default: %u)"), DEFAULT_ALERTS));
     strUsage += HelpMessageOpt("-alertnotify=<cmd>", _("Execute command when a relevant alert is received or we see a really long fork (%s in cmd is replaced by message)"));
     strUsage += HelpMessageOpt("-blocknotify=<cmd>", _("Execute command when the best block changes (%s in cmd is replaced by block hash)"));
+    if (showDebug)
+        strUsage += HelpMessageOpt("-blocksonly", strprintf(_("Whether to operate in a blocks only mode (default: %u)"), DEFAULT_BLOCKSONLY));
```

Comment:
This is very vague about what it does.

Reply:
I suggest pulling in https://github.com/bitcoin/bitcoin/pull/15990/commits/fa8ced32a60dea37ac169241cf9a1f708ef46c4b (notwithstanding the changes in option handling).

Reply:
Also, do we want this to be a non-debug option?